core/commands/generate_product_repositories_command.rb

```diff
@@ -593,8 +593,8 @@ def parse_repository(base_url, auth, key, product, *steps)
       next_releases = Workers.map(releases) do |release|
         begin
           links = get_directory_links(release[:url], auth)
-        rescue OpenURI::HTTPError, SocketError, Net::OpenTimeout => error
-          error_and_log("Unable to get information from link '#{release[:url]}', message: '#{error.message}'")
+        rescue OpenURI::HTTPError, SocketError, Net::OpenTimeout, Errno::ECONNRESET, OpenSSL::SSL::SSLError, RuntimeError => e
```

Comment:
You can limit rescue clause only to StandardError as we actually want to capture all possible errors here.